nss-copper-dragonflies/WorkforceManagement
# Brj Training Program Edit and Details

# Description

Added edit and details functionality to the training programs feature

Fixes # [Ticket #10](https://github.com/nss-copper-dragonflies/WorkforceManagement/issues/10)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch -a`
1. `git checkout brj-training-program-edit-and-details`
1. Click on your SQL server and indicate `yes to all` in order to get the data necessary to test this feature
1. Run the project
1. Click on the Training Programs tab
1. Click on the details button for the "How to Sell Cats" program
1. Ensure that details include multiple employees as attending the program
1. Click on the "Edit" button
1. Make changes
1. Upon being rerouted to the list ensure that your edits appear



# Checklist:
